src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,81 @@
+import PropTypes from "prop-types";
+import React, { useRef, useEffect } from "react";
+import styles from './styles.css';
+import Icon from '../Icon/Icon.jsx';
+
+const Dropdown = ({
+  children,
+  title,
+  handleDropdown,
+  show,
+  style,
+  onSelectItem,
+  closeOutsideClick,
+  ...props
+}) => {
+	const renderChildrenItems = () => {
+    return React.Children.map(children, (child, index) => {
+      return React.cloneElement(child, {
+      	style: style,
+      	onSelect: onSelectItem, 
+        itemindex: index,
+      });
+    });
+  };
+
+	function useOutsideAlerter(ref) {
```

Comment:
Your code is missing indentation in many parts, make sure to run lint scripts to get it fixed. It should be running automatically on pre-commit though. @tiagoalvesdulce do you know what is going on here?

Reply:
Hmmm that is strange. When I run `yarn lint` locally, I get a few errors from other components (ones not edited in this PR).